pkgs/os-specific/linux/kvms/default.nix

```diff
@@ -0,0 +1,13 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation ({
+
+    name = "kvms";
+
```

Comment:
Please add comment on how and why binary blob is built from https://github.com/jkrh/kvms